application/app/src/test/java/uk/nhs/digital/uec/api/controller/FuzzySearchControllerTest.java

```diff
@@ -131,15 +132,16 @@ public void getServicesByFuzzySearchTestValidationError() throws ValidationExcep
             NAME_PRIORITY,
             ADDRESS_PRIORITY,
             POSTCODE_PRIORITY,
-            PUBLIC_NAME_PRIORITY);
+            PUBLIC_NAME_PRIORITY,
+            null);
 
     // Assert
     final ApiValidationErrorResponse response =
         (ApiValidationErrorResponse) responseEntity.getBody();
 
     verify(mockValidationService, times(1)).validateSearchCriteria(searchCriteria);
     verify(mockFuzzyServiceSearchService, never())
-        .retrieveServicesByFuzzySearch(SEARCH_POSTCODE, searchCriteria);
+        .retrieveServicesByFuzzySearch(SEARCH_POSTCODE, searchCriteria, null);
```

Comment:
here

Reply:
These changes were already merged, this branch was rebased from master